text/0000-optimize-by-default.md

```diff
@@ -0,0 +1,166 @@
+- Start Date: 2015-03-10
+- RFC PR: (leave this empty)
+- Rust Issue: (leave this empty)
+
+# Summary
+
+Make rustc and cargo produce optimized binaries by default.
+
+# Motivation
+
+
+Optimizing by default is a safety & usability feature that everyone benefits
```

Comment:
I'll admit you have a solid point in that "unsafe" is a pretty loaded term in the Rust world; it usually means memory-unsafe.

If you wish, I could clarify in the RFC what I mean. Which change would you like to see? If you have a specific, "here's how I'd phrase it" suggestion, please provide it.